executor/write.go

```diff
@@ -812,11 +734,6 @@ func (e *LoadData) exec(ctx context.Context) (Row, error) {
 	return nil, nil
 }
 
-// Next implements the Executor Next interface.
-func (e *LoadData) Next(ctx context.Context) (Row, error) {
-	return e.exec(ctx)
-}
-
 // NextChunk implements the Executor NextChunk interface.
 func (e *LoadData) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
```

Comment:
we can remove `exec` and put the logical inside `NextChunk`

Reply:
```go
    if e.done {
        return nil
    }
    err := e.run(ctx)
    e.done = true
    return errors.Trace(err)
```
Is it?

Reply:
no, I mean we can remove function `exec` and put the codes of `exec` into `NextChunk`

Reply:
I mean I remove Next() and put that codes of `exec` into `NextChunk`.

Reply:
Only LoadData? Because it makes the code simple.